docs/source/overview.rst

```diff
@@ -256,8 +256,8 @@ We could also say that each point in the tagged data (e.g. a matrix of
 measurements) has a corresponding point in an input matrix.
 
 .. code-block:: python
-		
-		input_matrix = np.random.randn(data.shape)
+		a,b=data.shape
+		input_matrix = np.random.randn(a,b)
```

Comment:
actually I would prefer the previous version since it is more compact and also illustrates interactions better